README.md

```diff
@@ -72,8 +72,12 @@ app.on("token", async ({ token, octokit }) => {
 });
 
 require("http").createServer(getNodeMiddleware(app)).listen(3000);
-// can now receive user authorization callbacks at /api/github/oauth/callback
+// or `expressApp.use(getNodeMiddleware(app));` for Express apps
 ```
+## Endpoints
+
+- Redirect to authorization prompt at `/api/github/oauth/login`
+- Receive user authorization callbacks at `/api/github/oauth/callback`
```

Comment:
We document the endpoints at https://github.com/octokit/oauth-app.js#middlewares, maybe could just link to them? How about this?

```suggestion
// can now receive user authorization callbacks at /api/github/oauth/callback
// See all endpoints at https://github.com/octokit/oauth-app.js#middlewares
```

Reply:
That is a really good idea, it just took me sometime to fund the `/api/github/oauth/login` endpoint and  wasn't sure where to look.